src/layer/BingTiledImageLayer.js

```diff
@@ -103,12 +86,25 @@ define([
             this.topLevelTiles.push(this.createTile(null, this.levels.firstLevel(), 1, 1));
         };
 
+        BingTiledImageLayer.prototype.renderLogo = function (dc) {
+            // Assigning ScreenImage to logoImage in here due to circular dependency of
```

Comment:
Lazy object creation is a common pattern in WorldWind. This comment doesn't add anything.